filament/src/materials/sgsr/sgsr1.mat

```diff
@@ -22,8 +27,8 @@ material {
 
 vertex {
     void postProcessVertex(inout PostProcessVertexInputs postProcess) {
-        postProcess.normalizedUV = uvToRenderTargetUV(postProcess.normalizedUV);
```

Comment:
I don't know enough to say for sure, but just want to check that this is no longer necessary?

I see the same transform in many other material files.

Reply:
yeah I double checked, this is correct.  The postfx framework calculates` normalizedUV` for us as a convenience. Then we use it to calculate our variable, and in most case we need the `uvToRenderTargetUV` call. In the previous code we were using `normalizedUV` as a temporary, but this variable is only for our benefit.